pkg/scheduler/framework/interface.go

```diff
@@ -640,3 +640,12 @@ type PluginsRunner interface {
 	// RunPreFilterExtensionRemovePod calls the RemovePod interface for the set of configured PreFilter plugins.
 	RunPreFilterExtensionRemovePod(ctx context.Context, state *CycleState, podToSchedule *v1.Pod, podInfoToRemove *PodInfo, nodeInfo *NodeInfo) *Status
 }
+
+// ResourceNameQualifier abstracts helpers qualifying
+// of what kind a resource name is.
+type ResourceNameQualifier interface {
```

Comment:
-1

Why is this not in component-helpers?

Reply:
I will leave this for discussion. I don't have a strong opinion about where it should be.